cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -0,0 +1,454 @@
+{
+    "$schema": "http://json-schema.org/draft-03/schema#",
+    "description": "Record",
+    "type": "object",
+    "properties": {
+        "date_reprocessed": {
+            "type": "string"
+        },
+        "comment": {
```

Comment:
We use `note` to replace what was previously "Issues and Limitations", but we also have `comment`. I am wondering whether `comment` is needed, since the difference between these two terms is not obvious to an "outsider". I think it is only used in `atlas-higgs-challenge-2014.json`, so perhaps it wouldn't be difficult to kill it.

Reply:
👍 for merging `note` and `comment`. They are not used in the same record, so it'll be easy to merge them.  Would you like to retain `note` or the `comment` as the property name?

Reply:
I think I like `note` more

Reply:
OK, done.

Reply:
Can it be removed from this schema as well?

Reply:
Whoops! Done.